src/components/PostListRow.js

```diff
@@ -100,9 +113,13 @@ export const PostListRow = React.createClass({
         <View style={ postRowStyle }>
           {image}
           <View style={styles.articleContainer}>
-            <Text numberOfLines={1} style={styles.tags}>{tagsString}</Text>
             <Text style={styles.title}>{post.title}</Text>
             <Text numberOfLines={3} style={styles.teaser}>{post.teaser}</Text>
+            <View style={styles.articleInfo}>
+              <Text numberOfLines={1} style={styles.tags}>{tagsString}</Text>
+              <View style={{flex: 1}} />
+              <Text numberOfLines={1} style={styles.tags}>{dateString}</Text>
+            </View>
```

Comment:
Changes the format of the post list row to look like this:

![img1](https://cloud.githubusercontent.com/assets/9957046/18215177/f6233a76-711d-11e6-81b5-0cd966836a3d.png)